client/my-sites/plan-features/header.jsx

```diff
@@ -212,23 +256,27 @@ export class PlanFeaturesHeader extends Component {
 						<PlanPrice
 							currencyCode={ currencyCode }
 							rawPrice={ fullPrice }
-							isInSignup={ isInSignup }
+							isInSignup={ displayFlatPrice }
```

Comment:
Nit: if PlanPrice is only used here, we should rename `isInSignup` prop to `displayFlatPrice`